gofile-downloader.py

```diff
@@ -339,7 +339,9 @@ def _parseLinksRecursively(
                 "filename": data["name"],
                 "link": data["link"]
             }
-
```

Comment:
We're missing some spaces between the functions here, also we should use a more descriptive name.
```patch

-    def _listFiles(self) -> None:
+
+
+    def _printListFiles(self) -> None:
+        """
+        _printListFiles
+
+        Helper function to display a list of all files for selection.
+
+        :return:
+        """
+
         width: int = max(len(self._files_info[k]["filename"]) for k in self._files_info)
-        for index in self._files_info: _print(f"[{index}] -> {self._files_info[index]["filename"].ljust(width)} - {NEW_LINE}")
+
+        for index in self._files_info: _print(f"[{index}] -> {self._files_info[index]["filename"].ljust(width)}{NEW_LINE}")
+
```